src/main/java/com/zaxxer/hikari/pool/HikariPool.java

```diff
@@ -573,7 +573,7 @@ public Boolean call() throws Exception
 
             // failed to get connection from db, sleep and retry
             quietlySleep(sleepBackoff);
-            sleepBackoff = Math.min(connectionTimeout / 2, (long) (sleepBackoff * 1.3));
+            sleepBackoff = Math.min(connectionTimeout / 5, (long) (sleepBackoff * 1.3));
```

Comment:
Why 5?  With the minimum ``connectionTimeout`` of 1000ms this will generate a constant load of 5 connection attempts per second (200ms apart).  ``/ 2`` gives a much more reasonable retry value.